File: grade-simulator-service/Backend/models/Student.js
const Sequelize = require('sequelize');
const sequelize = require('../config/database');

const Student = sequelize.define('Students', {
  student_id: {
    type: Sequelize.STRING(10),
    primaryKey: true,
  },
  name: {
    type: Sequelize.STRING(100),
    allowNull: false,
  },
  email: {
    type: Sequelize.STRING(100),
    allowNull: false,
  },
  program_id: {
    type: Sequelize.STRING(10),
    allowNull: false,
  },
}, {
  timestamps: false,
  tableName: 'Students',
});

module.exports = Student;

File: grade-simulator-service/dummy_data/models/student.py
from sqlalchemy import Column, String, Integer
from db import Base

class Student(Base):
    __tablename__ = "Students"
    
    student_id = Column(String(10), primary_key=True)
    name = Column(String(100), nullable=False)
    email = Column(String(100), nullable=False)
    program_id = Column(String(10), nullable=False)
    
    def to_dict(self):
        return {
            "student_id": self.student_id,
            "name": self.name,
            "email": self.email,
            "program_id": self.program_id
        } 